src/agenda.rs

```diff
@@ -606,23 +606,17 @@ pub fn lang_planning<'a>() -> Box<dyn Action> {
 pub fn compiler_backlog_bonanza<'a>() -> Box<dyn Action> {
     Box::new(Step {
         name: "compiler_backlog_bonanza",
-        actions: vec![
-            Query {
-                repos: vec![
-                    ("rust-lang", "rust"),
-                ],
-                queries: vec![
-                    QueryMap {
-                        name: "tracking_issues",
-                        kind: QueryKind::List,
-                        query: Box::new(github::Query {
-                            filters: vec![("state", "open")],
-                            include_labels: vec!["C-tracking-issue"],
-                            exclude_labels: vec!["T-libs-api", "T-libs", "T-lang", "T-rustdoc"],
-                        }),
-                    },
-                ],
-            },
-        ],
+        actions: vec![Query {
+            repos: vec![("rust-lang", "rust")],
+            queries: vec![QueryMap {
+                name: "tracking_issues",
+                kind: QueryKind::List,
+                query: Box::new(github::Query {
+                    filters: vec![("state", "open")],
+                    include_labels: vec!["C-tracking-issue"],
+                    exclude_labels: vec!["T-libs-api", "T-libs", "T-lang", "T-rustdoc"],
+                }),
+            }],
+        }],
```

Comment:
this is just formatting? (do we have a mandatory rustfmt style on this repo?)

Reply:
looks like that is just formatting, I will happily revert this to how it was before. 

Not sure if there's a rustfmt style on this repo or not — is there some sort of config file that would indicate whether that's true or not? 

*Could* just be my VSCode settings doing their thing — will happily change them to make them stop auto-formatting everything if need be

Reply:
Reverted in cdc9231